pkgs/top-level/python-packages.nix

```diff
@@ -22660,6 +22660,13 @@ EOF
   voluptuous = callPackage ../development/python-modules/voluptuous { };
 
   pysigset = callPackage ../development/python-modules/pysigset { };
+
+  jinja2_pluralize = callPackage ../development/python-modules/jinja2_pluralize { };
```

Comment:
Attributes are roughly organized. Surely you can find a better place for these.

Reply:
What is criteria of "better place" ?
I don't understand why this place is bad.

Reply:
The attributes should be ordered by name.

Reply:
Should be order by derivation name or by attribute name?
I still does not understand what you want because the attribute set doesn't have lexicographical order...

If i put this attribute near line 9493 (near jinja package) will it be ok?